docs/contribute-to-romcal.md

```diff
@@ -125,3 +125,51 @@ _pass the `-t` flag and the path of the file_
 ## Coverage reporting
 
 A coverage report is generated for every core branch build and published to [gh-pages]().
+
+## Committing changes
+
+### Message Format
+
+When you are ready to commit your changes, your commit message should follow the conventional commit format
+
+General format/terminology:
+
+```
+<type>[optional scope]: <description>
+
+[optional body]
+
+[optional footer(s)]
+```
+
+Here are some guardrails to follow:
+
+When "type" one of the keys, the scopes allowed are as follows:
+
+- `ci` (updates to automation or the release process): No scope allowed
```

Comment:
In this list, I’d probably  use lowercase initial letter and append semicolon to each list item. Again, just a preferencd. :wink:

Reply:
this is markdown, not js :thinking:

Reply:
Yeah, it is MD, however:

- I personally don’t use semicolons after commands in JS (unless they are necessary);
- I treat MD files as text in articles/books, i.e. they should follow some style (as in _manual of style_, e.g. Chicago Manual of Style or MLA.

Therefore, I suggest to choose a style and use it consistently. I personally (and you don’t need to accept it) prefer ending lists with either a comma (unless a comma is used at least once in at least one of the list items) or a semicolon (which can be used even there is no comma in any of the list items).

Sorry, I am a _formalist_, as in _I love the form (in contrary to the content)_. Feel free to ignore my and keep it as is. :wink: